bugsnag/src/test/java/com/bugsnag/ThreadMetaDataTest.java

```diff
@@ -0,0 +1,185 @@
+package com.bugsnag;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.bugsnag.delivery.Delivery;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Map;
+
+public class ThreadMetaDataTest {
```

Comment:
It'd be good to add separate test methods for `clearThreadMetaData(String tabName)` and `clearThreadMetaData(String tabName, String key)`